qoffer-infrastructure/src/main/resources/offer-template/offer.html

```diff
@@ -15,27 +15,27 @@
     <div class="grid-container">
         <div class="header"><img src="offer_header.png"/></div>
         <div class="customer">
-            <div class="addressee">Dr. Max Mustermann</div>
-            <div class="description">Quantitative Biology Center</div>
-            <div class="street">Auf der Morgenstelle 10</div>
-            <div class="zipcode">72076</div>
-            <div class="town">Tübingen</div>
-            <div class="country">-</div>
+            <div id="cName">Dr. Max Mustermann</div>
+            <div id="cOrganisation">Quantitative Biology Center</div>
+            <div id="cStreet">Auf der Morgenstelle 10</div>
+            <div id="cPostalCode">72076</div>
+            <div id="cCity">Tübingen</div>
+            <div id="cCountry">-</div>
```

Comment:
I would prefer a more declarative naming of ids. Instead of `cXYZ` using `customerXYZ` would improve readability greatly.

Reply:
I agree, but this can go in the refactoring of the template imo

Reply:
added to trello card